Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -2082,6 +2133,13 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
       mcTrackLabelCursor(0, labelHolder.labelID, labelHolder.labelMask);
     }
   }
+
+  // Fill calo tables and if MC also the MCCaloTable, therefore, has to be after fillMCParticlesTable call!
+  if (mInputSources[GIndex::PHS] || mInputSources[GIndex::EMC]) {
+    fillCaloTable(caloCellsCursor, caloCellsTRGTableCursor, mcCaloLabelsCursor, bcsMap, recoData);
+  }
+
+  bcsMap.clear();
```

Comment:
I hope moving the `bcsMap.clear();` down under the filling of the MCParticle table is okay.
@ddobrigk do you know this or know who to ask?

Reply:
Has this been clarified?

Reply:
Sorry, I missed this (somehow it is still easier for me to miss this kind of notification). I don't really think this should be an issue but I suppose we could double-check with @ekryshen to be double-sure....

Reply:
I would better ask @nburmaso to be double-sure...

Reply:
Hey @nburmaso 
the `fillCaloTable` now also fills the MC Calo tables and needs the `mToStore` which is filled in `fillMCParticlesTable` if I understand the code correctly, so at least this function must be called before. But I can move it above `fillMCTrackLabelsTable`. In that case I would need to split the `if (mUseMC)`, one for `fillMCParticlesTable` and then one again for `fillMCTrackLabelsTable`

Reply:
@mhemmer-cern, sorry, I thought that mToStore is not used in fillCaloTable. Then it should be totally fine to leave it as it is now :)